projects/packages/my-jetpack/_inc/components/connection-status-card/index.tsx

```diff
@@ -63,7 +63,7 @@ const ConnectionListItem: ConnectionListItemType = ( {
 		<div className={ styles[ 'list-item' ] }>
 			<Text className={ clsx( styles[ 'list-item-text' ], statusStyles ) }>
 				{ icon && <Icon icon={ icon } /> }
-				{ text }
+				<span>{ text }</span>
```

Comment:
Was this for some sort of styling change? I don't see any CSS changes related to this so I just wanted to make sure it wasn't leftover from a previous implementation

Reply:
Yes, it was to appease the tests 😅 They were having trouble locating an element with the specified text on the screen, adding a wrapping element helped the matching

Reply:
Makes sense 😆